database/models/tool.ts

```diff
@@ -16,6 +16,7 @@ export interface ToolInterface {
     relatedToolsIDs: [Schema.Types.ObjectId];
     status: StatusType;
     editing: boolean;
+    createdBy: string[];
```

Comment:
when are there multiple "creators" of a tool?

Reply:
not totally sure if we have the use case of multiple creators of a tool- just followed the schema plan from the[ database schema on Notion](https://www.notion.so/uwblueprintexecs/Database-Schema-a256105a04c946f7a69d26613b2388bd)

Reply:
sounds good!